packages/babel-helpers/src/helpers/regeneratorRuntime.ts

```diff
@@ -561,6 +567,99 @@ export default function /* @no-mangle */ _regeneratorRuntime() {
       : false;
   }
 
+  function AsyncIterator(
```

Comment:
Doesn't this mean that `AsyncIterator` is inlined twice for all usages of the `regeneratorRuntime` helper? (one from here, one from `regeneratorAsyncGen`)

Reply:
Yes, this is a drawback, but we have already repeatedly inlined `regenerator`.

Reply:
I'm not sure I understand, when would it be inlined twice?

If we inject both `regeneratorRuntime` and `regenerator`, the first one will have a dependency on the second one without duplicating it.

Reply:
Oops, my mistake, I meant to say that there would be two versions of `regenerator.wrap`.